examples/06_datetime_encoder.py

```diff
@@ -0,0 +1,159 @@
+"""
+Handling datetime features with the DatetimeEncoder
+==========================================
+
+We illustrate here how to handle datetime features with the DatetimeEncoder. The DatetimeEncoder
+breaks down each datetime features into several numerical features, by extracting relevant information from the
+datetime features, such as the month, the day of the week, the hour of the day, etc. Used in
+the SuperVectorizer, which automatically detects the datetime features, the DatetimeEncoder allows
+to handle datetime features easily.
+"""
+
+import warnings
+warnings.filterwarnings("ignore")
+
+###############################################################################
+# Data Importing
+# --------------
+#
+# We first get the dataset.
+# We want to predict the NO2 air concentration in different cities, based on the date and the time of measurement.
+import pandas as pd
+data = pd.read_csv("https://raw.githubusercontent.com/pandas-dev/pandas/main/doc/data/air_quality_no2_long.csv")
+y = data["value"]
+X = data[["city", "date.utc"]]
+X
+
+###############################################################################
+# Creating encoders for categorical and datetime features
+# -------------------------
+from sklearn.preprocessing import OneHotEncoder
+from dirty_cat.datetime_encoder import DatetimeEncoder
+
+cat_encoder = OneHotEncoder(handle_unknown="ignore")
+datetime_encoder = DatetimeEncoder(add_day_of_the_week=True,  # The day of the week is probably relevant
+                                   extract_until="minute")  # We're probably not interested in seconds and below
+
+from sklearn.compose import make_column_transformer
+
+datetime_columns = ["date.utc"]
+categorical_columns = ["city"]
+
+encoder = make_column_transformer((cat_encoder, categorical_columns),
+                                  (datetime_encoder, datetime_columns),
+                                  remainder="drop")
+
+
+###############################################################################
+# Transforming the input data
+# ----------------------------
```

Comment:
```suggestion
# ..........................
```
Decreasing the number of top-level sections (to make the table of content in the side bar more readable)